PivotGrid: add Overview (#7107)
DevExpress/devextreme-documentation: concepts/05 UI Components/PivotGrid/02 Overview/10 Key Features.md

- **Summary Calculation Modes**    
PivotGrid displays [value summaries](/Documentation/Guide/UI_Components/PivotGrid/Summaries/). Choose between [predefined summaries](/Documentation/Guide/UI_Components/PivotGrid/Summaries/#Predefined_Summaries), [aggregate](/Documentation/Guide/UI_Components/PivotGrid/Summaries/#Custom_Aggregate_Function) values, or a implement a [custom summary](/Documentation/Guide/UI_Components/PivotGrid/Summaries/#Custom_Summaries).

- **Customizable Layout**    

> You can enable or disable multiple PivotGrid elements. More details are available in the [Visual Elements](/Documentation/Guide/UI_Components/PivotGrid/Visual_Elements/) article.

Reviewer: ```suggestion
You can enable or disable multiple PivotGrid elements. Review the [Visual Elements](/Documentation/Guide/UI_Components/PivotGrid/Visual_Elements/) article for more information.
```

- **Grouping, Sorting, and Filtering**    
PivotGrid [groups data](/Documentation/Guide/UI_Components/PivotGrid/Grouping/) for header items and fields. [Sort](/Documentation/Guide/UI_Components/PivotGrid/Sorting/) or [filter](/Documentation/Guide/UI_Components/PivotGrid/Filtering/) fields through the UI or programmatically.

- **Export to Excel**    
[Export PivotGrid data to Excel](/Documentation/Guide/UI_Components/PivotGrid/Export/) in XLSX or CSV format and customize the exported cells.

- **Chart Integration**    

> PivotGrid handles large numeric datasets, which can be hard to interpret. For detailed data visualization, [use the Chart component](/Documentation/Guide/UI_Components/PivotGrid/Integration_with_Chart/) that integrates with PivotGrid.

Reviewer: ```suggestion
PivotGrid handles large numeric datasets that can be hard to interpret. For a detailed data visualization, [use the Chart component](/Documentation/Guide/UI_Components/PivotGrid/Integration_with_Chart/) that integrates into PivotGrid.
```
